analyses/verbose/types.go

```diff
@@ -29,7 +29,7 @@ type DependenciesType struct {
 	Transitives                  []DependenciesType  `json:"transitives"`
 	LatestVersion                string              `json:"latest_version"`
 	RecommendedVersion           string              `json:"recommended_version"`
-	CommonlyKnownVulnerabilities []VulnerabilityType `json:"commonly_known_vulnerabilities"`
+	CommonlyKnownVulnerabilities []VulnerabilityType `json:"publicly_available_vulnerabilities"`
```

Comment:
Should we change the field name as well? @yzainee ?
CommonlyKnownVulnerabilities => PubliclyAvailableVulnerabilities?

Reply:
it wouldnt matter much in the end. will leave it to @deepak1725

Reply:
@arajkumar its internal usage only.. I feel it's okay to keep it as it is. 
Please let me know if you have strong opinions regarding that.

Reply:
Anyway, included that update also
 > CommonlyKnownVulnerabilities => PubliclyAvailableVulnerabilities?